scripts/python/tao_build.py

```diff
@@ -170,35 +168,6 @@ def _action_env(key, value, cmd="build"):
     configure_compiler_platform_alibaba(root, args)
     logger.info("Stage [configure compiler] success.")
 
-def config_blade_gemm(root, args):
```

Comment:
just ensure blade_gemm built here is never used ?

Reply:
This is for `tao_compiler` build, I have checked with @minminsun this is no not needed.